### PR TITLE
VIH-5255 Fix linting issues

### PR DESCRIPTION
**Pull request by npoxon**
### Change description ###
- Fix linting issues

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: TestWeb.AcceptanceTests/Steps/CreateHearingsSteps.cs [hunk 1]
```diff
@@ -68,18 +68,18 @@ private void GetTheHearingNames()
         {
             var summary = _browser.Driver.WaitUntilVisible(CreateHearingPage.SummaryTextfield).GetProperty("value");
             summary = summary.Replace("\r\n", ".");
-            var sentences = summary.Split(new[] { '.' });
+            var sentences = summary.Split('.', ':', '\'');
```
Comment (levibutler) [inline TestWeb.AcceptanceTests/Steps/CreateHearingsSteps.cs:71]: i didnt know Split took params char[] :) thanks

File: TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs [hunk 2]
```diff
@@ -1,76 +1,97 @@
...
+            azureAdConfiguration.Authority.Should().NotBeNullOrEmpty();
+            azureAdConfiguration.ClientId.Should().NotBeNullOrEmpty();
+            azureAdConfiguration.ClientSecret.Should().NotBeNullOrEmpty();
+            azureAdConfiguration.TenantId.Should().NotBeNullOrEmpty();
         }
 
-        protected async Task SendGetRequest(string uri)
+        protected async Task SendGetRequestAsync(string uri)
         {
-            using var client = CreateNewClient();
```
Comment (levibutler) [inline TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs:42]: +1 for using c# 8 using declarations :)

File: TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs [hunk 3]
```diff
@@ -80,5 +101,11 @@ protected void VerifyResponse(HttpStatusCode statusCode, bool isSuccess)
             Response.StatusCode.Should().Be(statusCode);
             Response.IsSuccessStatusCode.Should().Be(isSuccess);
         }
+
+        [OneTimeTearDown]
+        public void OneTimeTearDown()
+        {
+            _server.Dispose();
```
Comment (levibutler) [inline TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs:108]: is there a case for a null check before calling dispose?

**Review by levibutler — APPROVED** [review]
a lovely pr :)

File: TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs (hunk 3 above)
Comment (npoxon) [reply TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs:108]: ah yep, let me add it

Comment (npoxon) [reply TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs]: Cheers!

File: TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs (hunk 2 above)
Comment (npoxon) [reply TestWeb.IntegrationTests/Controllers/ControllerTestsBase.cs:42]: *tips cowboy hat*

**Pull request merged**